packages/js-drive/lib/abci/handlers/proposal/processProposalFactory.js

```diff
@@ -55,6 +56,7 @@ function processProposalFactory(
         version,
         time,
         proposerProTxHash: Buffer.from(proposerProTxHash),
+        proposedAppVersion,
```

Comment:
We need to make sure that it can't be nil. @shotonoff

Reply:
It can't be nil, just confirmed.